### ISSUE TITLE
Cannot set API key on development UI

### ISSUE THREAD

**Issue by mzperix** [meilisearch/MeiliSearch#1898]
**Describe the bug**
I run the the v0.24.0rc2 docker image and go to `/` (the default ui for testing the indexes) and I cannot set the API key, and hence cannot see any indexes, cannot search, etc. (With normal curl requests, the image does work, it just seems the UI doesn't understand a key should be set -- see screenshots).

**To Reproduce**
```
docker run --rm -it -p 7700:7700 -e MEILI_MASTER_KEY=test getmeili/meilisearch:v0.24.0rc2
```
On the UI (at `localhost:7700`) I cannot set the key, the input bar does not show up claiming "a key has not been set".

**Expected behavior**
I can set the key on the ui with earlier, 0.22.0 version.
```
docker run --rm -it -p 7700:7700 -e MEILI_MASTER_KEY=test getmeili/meilisearch:v0.22.0
```

**Screenshots**
`0.24.0rc2`: ![image](https://user-images.githubusercontent.com/2047797/141436485-dafee991-66a2-47ab-9472-db2b195cc080.png)

`0.22.0`: 
![image](https://user-images.githubusercontent.com/2047797/141436579-c4df6188-4643-4256-8d63-abfb76b0a943.png)

**MeiliSearch version:** [v0.24.0rc2]

**Comment by curquiza** [comment]
Hello @mzperix 

thanks a lot for this report! We will change this before the official release!

**Comment by curquiza** [comment]
Fixed by #1904
